libkatana_python_native/src/EntityTypeManager.cpp

```diff
@@ -35,11 +35,23 @@ katana::python::EntityType::Make(
   }
 }
 
```

Comment:
I was thinking about this. On one hand, I think it's silly to duplicate information from the directories in the filename. On the other hand it might be better to be explicit in the file name because people don't pay attention.
You are right that we should have matching header and source names. The questions is whether they should be `SubsystemPython` or just `Subsystem`. For example, should we have `python/EntityTypeManagerPython.{cpp,h}` or `python/EntityTypeManager.{cpp,h}`.

Reply:
I would say explicit so `EntityTypeManagerPython.{cpp, h}`. I think some people can overlook that `python/` easily when searching.

Reply:
Discussed in slack: This will be handled in another PR.